src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/mock-subcategories.ts

```diff
@@ -0,0 +1,382 @@
+export const mockSubcategory = [
+  {
+    public_name: 'Afval',
```

Comment:
Better to put in a __test__ folder, as we do on other places too.

Reply:
Did it, thought we wanted to move away from the _test_folders.